wis2box-ctl.py

```diff
@@ -184,6 +184,7 @@ def make(args) -> None:
         _ = run(args, split('docker ps -a -q'), asciiPipe=True)
         run(args, split(f'docker rm {_}'))
     elif args.command == "restart":
+        print(f"Restarting {containers}")
```

Comment:
Remove print statement.

Reply:
That shouldn't be there, I appear to have contaminated my main branch. The latest commit should undo that.